frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/LinkedReleasesAndProjectsAwarePortlet.java

```diff
@@ -91,17 +95,18 @@ protected void putDirectlyLinkedReleasesInRequest(PortletRequest request, Projec
         request.setAttribute(RELEASE_LIST, linkedReleases);
     }
 
-    protected void putLinkedProjectsInRequest(PortletRequest request, Project project, User user) {
-        putLinkedProjectsInRequest(request, project, Function.identity(), user);
+    protected List<ProjectLink> createLinkedProjects(Project project, User user) {
```

Comment:
Ok, one can do this. But is there a deeper reason for abandoning `putLinkedProjectsInRequest`?  Is it only to have access to `mappedProjectLinks` in line 766 of the `ProjectPortlet`?  Wouldn't it be more readable to have a `createLinkedProjects` function as well as a `putLinkedProjectsInRequest` function? Then one wouldn't have to replace every occurrence of `putLinkedProjectsInRequest` by two lines all over the code.